python/src/CleaningStrategy_doc.i.in

```diff
@@ -139,10 +151,70 @@ getMaximumSize"
 
 Parameters
 ----------
-factor : float
+significanceFactor : float
     Value of the significance factor.
 
 See also
 --------
 getSignificanceFactor"
 
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::CleaningStrategy::setVerbose
+"Accessor to the verbose.
+
+Parameters
+----------
+verbose : bool
+    Enable the online monitoring of the current basis updates or not.
+
+See also
+--------
+getVerbose"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::CleaningStrategy::updateBasis
+"Update the basis for the next iteration of approximation.
+
+In this strategy, the *residual* and the *relativeError* input arguments
+are ignored.
+
+Parameters
+----------
+alpha_k: sequence of floats
```

Comment:
we need spaces on both sides of :

Reply:
It subtle, but indeed, the "sequence of floats" is bold instead of bold _and_italic.
```suggestion
alpha_k : sequence of floats
```